operators/src/processing/expression/mod.rs

```diff
@@ -52,6 +53,7 @@ pub struct ExpressionParams {
 pub type Expression = Operator<ExpressionParams, ExpressionSources>;
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
+#[allow(clippy::unsafe_derive_deserialize)]
```

Comment:
why is this needed? there is no unsafe method here

Reply:
I'm not quite sure… Clippy warns… it might be a glitch:

[473daf5](https://github.com/geo-engine/geoengine/pull/559/commits/473daf56945b1a5972eafc33cae2918a33a9d531)